ts/Dashboards/Board.ts

```diff
@@ -662,15 +662,15 @@ namespace Board {
          *
          * Try it:
          *
-         * {@link https://jsfiddle.net/gh/get/library/pure/highcharts/highcharts/tree/master/samples/dashboards/demo/component-highcharts/  | Highcharts component}
+         * {@link https://www.highcharts.com/samples/embed/dashboards/components/component-highcharts | Highcharts component}
```

Comment:
Sorry for being the samples nitpicker today, but here they should probably be without `embed` or `nonav`

Reply:
So what the URL should be? 😅

Reply:
I think we should keep jsFiddle link 😅 Just like 99.99% (there is one exception) Highcharts API links redirect to jsFiddle, not to our demo pages. Unless there's a plan to change that? 🤔

Reply:
If we want to use highcharts.com/samples it should be https://www.highcharts.com/samples/dashboards/components/component-highcharts. (they're pretty similar, but `/embed` has some extra magic for being placed in iframes)

Reply:
Ok, let's just go with jsfiddle then 😇